tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -6905,19 +6907,45 @@ TEST_F(OpConverterTest, ConvertResize) {
   {
     // output dimension is a tensor, should fail.
     Reset();
-    NodeDef node_def =
-        MakeResizeNodeDef<ops::ResizeBilinear>("my_resize", DT_FLOAT, true);
+    NodeDef node_def = MakeResizeNodeDef<ops::ResizeBilinear>(tf_type_,
+                                                              /*align_corners=*/
+                                                              true);
     AddTestTensor("input", {1, 2});
     AddTestTensor("size", {1, 2});
     RunValidationAndConversion(
         node_def, error::UNIMPLEMENTED,
         "The input \"size\" for ResizeBilinear must be a "
         "constant, at my_resize");
   }
-  TestConvertResize<ops::ResizeBilinear, DT_FLOAT>(this);
-  TestConvertResize<ops::ResizeBilinear, DT_HALF>(this);
-  TestConvertResize<ops::ResizeNearestNeighbor, DT_FLOAT>(this);
-  TestConvertResize<ops::ResizeNearestNeighbor, DT_HALF>(this);
+
+  std::vector<ResizeTestParams> params{
+    {
+        /*input_dims=*/{1, 1, 2, 1},       // N, H, W, C
+        /*output_resize_dims=*/{2, 3},  // H_out, W_out
+        /*input_values=*/{2.0f, -1.0f},
+        /*align_corners=*/false,
+        /*expected_output_dims=*/{1, 2, 3, 1},  // N, H, W, C
+        /*expected_nearest_output_values=*/{2.0f, 2.0f, -1.0f, 2.0f, 2.0f, -1.0f},
+        /*expected_bilinear_output_values=*/{2.0f, 0.f, -1.0f, 2.0f, 0.f, -1.0f},
```

Comment:
Can you split the lines so that they are < 80 chars?

Reply:
Done